File: src/main/scala/analysis/session/bean/Top10Category.scala
package analysis.session.bean

/**
 * 品类Top10表
 * @param taskid
 * @param categoryid
 * @param clickCount
 * @param orderCount
 * @param payCount
 */
case class Top10Category(taskid:String,
                         categoryid:Long,
                         clickCount:Long,
                         orderCount:Long,
                         payCount:Long)

File: src/main/java/com/jxust/entity/Top10Category.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Top10Category implements Serializable {
    private String taskid;

    private Long categoryid;

    private Long clickcount;

    private Long ordercount;

    private Long paycount;

    private static final long serialVersionUID = 1L;
}